libs/platform/src/lib/components/form/form-group/form-group.component.html

```diff
@@ -3,38 +3,35 @@
 </form>
 
 <ng-template #body>
-    <!--  Render TOP Zone-->
-    <ng-container *ngTemplateOutlet="zone; context: { $implicit: tZone, title: topTitle }"> </ng-container>
-
-    <!--  Render MAIN Zone in which we usually support LEFT and RIGHT column -->
-    <ng-container *ngTemplateOutlet="zone; context: { $implicit: mZone, title: mainTitle, multiColumn: true }">
+    <!--  Render fields and field groups with assigned columns -->
+    <ng-container *ngTemplateOutlet="fields; context: { $implicit: formRows, title: mainTitle }">
     </ng-container>
-
-    <!--  Render BOTTOM Zone-->
-    <ng-container *ngTemplateOutlet="zone; context: { $implicit: bZone, title: bottomTitle }"> </ng-container>
 </ng-template>
 
-<ng-template #zone let-fields let-multiColumn="multiColumn" let-title="title">
-    <div
-        *ngIf="fields"
-        [formGroup]="formGroup"
-        [ngClass]="{ 'is-editable': editable }"
-        class="fd-section fd-section--no-border fd-has-margin-regular"
-    >
-        <div *ngIf="title" class="fd-section__header">
-            <h3 class="fd-section__title">{{ title }}</h3>
-        </div>
-
-        <div *ngIf="mZone" class="row start-xs">
-            <div
-                *ngFor="let f of fields; trackBy: trackByFieldName"
-                [ngClass]="f.styleClass"
-                [style.order]="f.rank"
-                class="col-xs-12"
-                fd-form-set
-            >
-                <ng-container *ngTemplateOutlet="f?.renderer"></ng-container>
-            </div>
+<ng-template #fields let-rows let-title="title">
+    <div *ngIf="title" class="fd-form-header">
+        <span class="fd-form-header__text">{{ title }}</span>
+    </div>
+    <div class="fd-container fd-form-layout-grid-container">
+        <div class="fd-row">
+            <ng-container *ngFor="let row of rows | keyvalue; trackBy: trackByFn">
+                <div class="fd-form-group__header" *ngIf="row.value.label">
+                    <h1 class="fd-form-group__header-text">{{ row.value.label }}</h1>
+                </div>
+                <ng-container *ngFor="let r of isFieldGroupRow(row) | keyvalue; trackBy: trackByFn">
+                    <div class="fd-form-group fd-col__form-group fd-col" [ngClass]="xlCol">
+                        <div class="fd-form-item fd-row__form-item fd-row">
+                            <div
+                                *ngFor="let f of r.value; trackBy: trackByFieldName"
```

Comment:
`f or r.value`, no idea what this means. please use a bit more descriptive names for variables